docs/migration.md

```diff
@@ -0,0 +1,128 @@
+# Migrate from PostgreSQL Community to Percona Distribution for PostgreSQL 
+
+
+Percona Distribution for PostgreSQL provides the PostgreSQL database and the tools, add-ons, and extensions in a package that is guaranteed to work together and is easy to deploy.
+
+We encourage users to migrate from their PostgreSQL deployments to Percona Distribution for PostgreSQL. This document provides the migration instructions. 
+
+Depending on your business requirements, you may migrate to Percona Distribution for PostgreSQL either [on the same server](#migrate-on-the-same-server) or [onto a different one](#migrate-on-a-different-server). 
+
+## Migrate on the same server
+
+=== "On Debian and Ubuntu Linux"
+
+     1. Back up your data. The backup process is out of scope of this document. You can use `pg_dumpall` or other tools of your choice. 
```

Comment:
I would add a note about *why* we suggest creating a backup here. Is it just a precaution? Or do we need the backup for the actual migration process?

Reply:
Updated